src/CMakeLists.txt

```diff
@@ -145,6 +148,8 @@ ENDIF()
 
    ADD_LIBRARY(qmcutil ${UTILITIES})
 
+   ADD_EXECUTABLE(test_utilities test/catch-main.cpp Utilities/test/test_PrimeNumberSet.cpp)
```

Comment:
In QMCPACK, the subdirectory is named 'tests'.   Not a big deal, but it might be good to be consistent.

Reply:
I will fix that.

Reply:
Please put `test/catch-main.cpp` to `Utilities/catch-main.cpp` this is what I asked earlier.
And renaming `Utilities/test` to `Utilities/tests` probably is what @markdewing means.